cluster-scope/base/clusterloggings/clusterLogging.yaml

```diff
@@ -17,10 +17,12 @@ spec:
     elasticsearch:
       nodeCount: 3
```

Comment:
change to  nodeCount: **1**

Tere is no need for 3 ES nodes at this point. This is aligned with "ZeroRedundancy"

Reply:
@4n4nd thanks for the support from my PoV if this can merge post this small fix 👍

Reply:
updated